From 0be91378e26a54cb4ef469290bc960d383ef2898 Mon Sep 17 00:00:00 2001
From: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date: Mon, 4 Aug 2014 17:13:56 +0200
Subject: [PATCH] beaglebone: use the am335x_boneblack u-boot configuration

Using the am335x_boneblack configuration for u-boot allows to save the
environment to the emmc. Else, it is not possible to save the
environment at all.

Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
---
 conf/machine/beaglebone.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/machine/beaglebone.conf b/conf/machine/beaglebone.conf
index 8a83a1aeb569..0b4dce3ad647 100644
--- a/conf/machine/beaglebone.conf
+++ b/conf/machine/beaglebone.conf
@@ -11,4 +11,4 @@ IMAGE_FSTYPES += "tar.gz"
 
 SERIAL_CONSOLE = "115200 ttyO0"
 
-UBOOT_MACHINE = "am335x_evm_config"
+UBOOT_MACHINE = "am335x_boneblack"
-- 
1.9.1

